### PR TITLE
fix: add OnConnectionEvent impl

### PR DESCRIPTION
**Pull request by CodingSinger**
Change-Id: I5a7bae956131c0beac407564f28d7ceb9ef9534c

### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/mock/network.go [hunk 1]
```diff
@@ -1,19 +1,17 @@
 // Code generated by MockGen. DO NOT EDIT.
-// Source: ../../../api/network.go
+// Source: /Users/zhengzechao/go/src/mosn.io/mosn/vendor/mosn.io/api/network.go
```
Comment (taoyuanyuan) [inline pkg/mock/network.go:2]: zhengzechao  。。

Comment (CodingSinger) [reply pkg/mock/network.go]: 擦，为啥还带完整路径

File: pkg/network/connection.go [hunk 2]
```diff
@@ -201,6 +201,13 @@ func (c *connection) SetIdleTimeout(readTimeout time.Duration, idleTimeout time.
 	c.newIdleChecker(readTimeout, idleTimeout)
 }
 
+func (conn *connection) OnConnectionEvent(event api.ConnectionEvent) {
+	log.DefaultLogger.Debugf("[network] receive new connection event %s, try to handle", event)
```
Comment (wangfakang) [inline pkg/network/connection.go:205]: maybe it's better that add a loglevel check before call `Debugf`.

**Review by taoyuanyuan — APPROVED** [review]
LGTM

**Review by wangfakang — APPROVED** [review]
LGTM.

**Pull request merged**